css/index.css

```diff
@@ -1,3 +1,300 @@
-/* Use your own code or past solution for Great Idea Web Page CSS here! */
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
 
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+body{
+	
+	margin: 3% 3% 3% 3%;
+	
+}
+
+section{
+	
+	
+
+
+}
+
+.block{
+
+	margin: 15px;
+	
+}
+.block p{
+	
+}
+
+h4{
+	font-size: 18px;
+	font-weight: bold;
+}
+
+header{
+	display: flex;
+	height: 5%;
+	justify-content: space-between;
+	
+}
+		
+	header nav{
+		display: flex;
+		
+		
+	}
+		header nav a{
+			margin:20px 25px 25px 25px;
+			color:black;
+			text-decoration-line: none;
+			justify-content: space-evenly;
+		}
+	header a{
+		display: flex;				
+	
+		align-items: center;
+		width: 20%;
+					
+		}
+	
+
+
+#bar1{
+	display: flex;
+	justify-content: space-around;
+	height: 35%;
```

Comment:
Removing the `height` will allow more room for your header on desktop. Also adding `margin-bottom` will give some room between the image and the next section.